.ddev/.typo3-setup/scripts/.utils.sh

```
@@ -0,0 +1,311 @@
+  message blue "Pre Setup for TYPO3 $VERSION"
+  install_start
+  message blue "Install Composer Packages"
+}
+
+# Function to perform post-setup tasks for TYPO3 installation.
+# It changes the current directory to the base path, sets the TYPO3 installation database name,
+# and calls the appropriate post-setup function based on the TYPO3 version.
+# After that, it imports data and updates TYPO3.
+function post_setup() {
+  message blue "Post Setup for TYPO3 $VERSION"
+  cd $BASE_PATH
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)